### PR TITLE
Shut down testing server in Travis

### PR DESCRIPTION
**Pull request by idealadarsh**
Done according to this task: https://codein.withgoogle.com/dashboard/task-instances/5592337747017728/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,11 +9,14 @@ before_script:
   - pip install numpy
   - pip install opencv-python
   - pip install -r requirements.txt
-  - python origami-lib/test/server_textimageinput.py &
-  - sleep 1
+  - |
+    python origami-lib/test/server_textimageinput.py &
+    SERVER_PID=$!
```
Comment (AvaisP) [inline .travis.yml]: Do we not need sleep anymore?

File: .travis.yml [hunk 2]
```diff
@@ -9,11 +9,14 @@ before_script:
...
 
 script:
   - cd origami-lib/test/
   - pytest
+  - echo $SERVER_PID
```
Comment (AvaisP) [inline .travis.yml]: echo isn't needed right?

File: .travis.yml (hunk 1 above)
Comment (idealadarsh) [reply .travis.yml]: No we don't.

File: .travis.yml (hunk 2 above)
Comment (idealadarsh) [reply .travis.yml]: It's not necessary but I used it just to show that server pid has a value and the process which is killed is the same.
Should I remove it?

Comment (AvaisP) [reply .travis.yml]: Yeah, remove it. It is a good idea to print variables while debugging but we dont need to have this in the production system  :smile:

**Comment by AvaisP** [comment]
Great job!

**Comment by idealadarsh** [comment]
@AvaisP Thank you. :)

**Pull request merged**